pkgs/by-name/ep/epy/package.nix

```
@@ -0,0 +1,26 @@
+{ lib, python3Packages, fetchPypi }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "epy";
+  version = "2023.6.11";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "epy_reader";
+    hash = "sha256-gel503e8DXjrMJK9lpAZ6GxQsrahKX+SjiyRwKbiJUY=";
+  };
+
+  nativeBuildInputs = [ python3Packages.setuptools ];
+
+  # Project has no tests
+  pythonImportsCheck = false;
```

```suggestion
  pythonImportsCheck = [
    "epy_reader.cli"
    "epy_reader.reader"
  ];
```
based on https://github.com/wustho/epy/blob/6b0e9fe0773f05fdf844b574f0f28df3961f60ab/src/epy_reader/__main__.py#L5-L6

---

Manually tested and seems the packages are possible to import.